test/SlowTests/Server/Documents/Patching/PatchDebugMode.cs

```diff
@@ -0,0 +1,656 @@
+﻿using System;
+using FastTests;
+using Raven.Client.Documents.Operations;
+using Sparrow.Json;
+using Xunit;
+using Xunit.Abstractions;
+using PatchRequest = Raven.Client.Documents.Operations.PatchRequest;
+
+namespace SlowTests.Server.Documents.Patching
+{
+    public class PatchDebugMode : RavenTestBase
+    {
+        public PatchDebugMode(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public void DebugResultsWhenLoadingData()
+        {
+            using (var store = GetDocumentStore())
+            {
+                var baseline = DateTime.Today;
+                
+                using (var session = store.OpenSession())
+                {
+                    session.Store(new Car { Model = "Golf" }, "cars/1");
+                    session.CountersFor("cars/1").Increment("Counter1", 100);
+                    session.TimeSeriesFor("cars/1", "TimeSeries1").Append(baseline.AddMinutes(1), new[] { 11d, 22d }, "tag1");
+                    session.SaveChanges();
+                }
+                
+                var reqEx = store.GetRequestExecutor();
+                using (reqEx.ContextPool.AllocateOperationContext(out var context))
+                {
+                    PatchOperation operation = new PatchOperation(id: "cars/1", changeVector: null, patch: new PatchRequest {
+                        Script = @"
+                                    load('cars/1');
+                                    load('cars/2');
+
+                                    counter('cars/1', 'Counter1');
+                                    counter('cars/1', 'Counter2');
+
+                                    timeseries(this, 'TimeSeries1').get();
+                                    timeseries(this, 'TimeSeries2').get();
+                                 "
+                        }, patchIfMissing: null);
+
+                    var cmd = operation.GetCommand(store, store.Conventions, context, reqEx.Cache, true, true);
+                    store.Commands().Execute(cmd);
+                    
+                    if (cmd.Result.Debug.TryGet("Actions", out BlittableJsonReaderObject actions) == false)
```

Comment:
You can use `Assert.True(cmd.Result.Debug.TryGet("Actions", out BlittableJsonReaderObject actions), "<Actions> section not found")`

Reply:
done for all places - thanks